### DISCUSSION TITLE
Is it possible to not include some (or all) default toolchain sdk includes/links?

### DISCUSSION THREAD

**Discussion by qudix** [xmake-io/xmake#2598] [Q&A]
For example, I have a project that contains its own standard library and as such don't need most of the default sdk includes that come with MSVC.

![image](https://user-images.githubusercontent.com/17361645/180614732-17f5db55-bb9e-472a-8c88-1c01e2edce21.png)

![image](https://user-images.githubusercontent.com/17361645/180614749-977cde46-c0be-4076-9199-1c2705175ccd.png)

**Comment by waruqi** [comment]
vs generator? vsxmake generator?

**Reply by qudix** [reply]
vsxmake in this case
